canbench-bin/tests/tests.rs

```diff
@@ -44,19 +44,19 @@ Benchmark: no_changes_test
 
 Summary:
   instructions:
-    changed: 0, unchanged: 1, new: 0, total: 1
-    change   : [min 0 | med 0 | max 0]
-    change % : [min 0% | med 0% | max 0%]
+    counts:   [total 1 | new 0 | improved 0 | regressed 0 | unchanged 1]
+    change:   [min 0 | med 0 | max 0]
+    change %: [min 0% | med 0% | max 0%]
```

Comment:
Is there an easy way to fix these tests after making formatting changes like this, or do you have to manually copy/paste to update the tests?

If you need to do it manually, we might want to consider using [insta](https://github.com/mitsuhiko/insta) to make it easier.

Reply:
yeah, unfortunately I had to do it manually.
thank you for suggesting insta, I'll try to use it in a follow-up PR.

Reply:
UPD: instead of using insta I decided to store golden files in plain text for better clarity, which allows to see git diffs. To overwrite expected files you just run cargo test with a feature flag `overwrite', see https://github.com/dfinity/canbench/pull/101.